client/Lykke.Service.BlockchainSettings.Client/CacheInvalidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Lykke.Service.BlockchainSettings.Client.Attributes;
+
+namespace Lykke.Service.BlockchainSettings.Client
+{
+    public class CacheInvalidator
+    {
+        private readonly object _locker;
+        private readonly IBlockchainSettingsClient _client;
+        private IEnumerable<(MethodInfo, InvalidateableClientCachingAttribute)> _methodAttributeTuples;
+
+        public CacheInvalidator(IBlockchainSettingsClient client)
+        {
+            _client = client;
+
+            var methods = typeof(IBlockchainSettingsClient).GetMethods(BindingFlags.Instance | BindingFlags.Public);
```

Comment:
Interface contains only instance public methods, so BindingFlags could be omitted.

Reply:
deleted